gateway/src/apicast/policy/nginx_metrics/nginx_metrics.lua

```diff
@@ -9,6 +9,13 @@ local select = select
 local find = string.find
 local pairs = pairs
 
+-- extended_metrics is a variable used to report multiple labels in some
+-- metrics. This can be useful in small environements, but can be problematic
+-- for large users due this can create a large matrix of metrics.
+-- More info about this can be found in Prometheus doc:
+-- https://prometheus.io/docs/practices/naming/#labels
+local extended_metrics = resty_env.get('APICAST_EXTENDED_METRICS')
```

Comment:
There is `resty_env.enabled` that will do automatic conversion of `true/false/1/0/...`. https://github.com/3scale/lua-resty-env#enabled